### PR TITLE
this is soo needed

### PR DESCRIPTION
**Pull request by CASPERg267**

### CODE REVIEW

**Comment by mechite** [comment]
This does not contribute anything useful to the project as the possible category values can only be as seen in that if statement. The formatting change above where you added a line break to the if statement was likely incidental as you would've probably cloned it before that had changed.

If there is a specific reason why you made this change, ie there is some reason why the values would only match case insensitive, please provide it and maybe make changes in the other files where these kind of assertions are made, such as the feed handler.

tl;dr: From my limited knowledge of this codebase, I can safely make the assumption that this change has no effect, if anything a negative effect.

**Comment by CASPERg267** [comment]
about the formatting maybe i wasn't on latest, i didn't mean that
and i made changes to the match case to be more specific cause it didn't see my owner category because its "Owner" instead of owner, so this works and it detects it to make the cmd Handler more efficient

**Review by breftejk — APPROVED** [review]

**Review by mohaidarus — APPROVED** [review]
Would be useful as not everyone uses lowercase to name their categories
